src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -2361,6 +2361,76 @@ class LedgerRPC_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testLedgerEntryMPT()
+    {
+        testcase("ledger_entry Request MPT");
+        using namespace test::jtx;
+        using namespace std::literals::chrono_literals;
+        Env env{*this};
+        Account const alice{"alice"};
+        Account const bob("bob");
+
+        MPTTester mptAlice(env, alice, {.holders = {&bob}});
+        mptAlice.create(
```

Comment:
Does it make sense to create two tokens by the same issuer? We should be able to get both providing the issuer account, right?

Reply:
Could you clarify? ledger_entry returns only one ledger object at a time, so I think one token is sufficient for testing.